usr/src/lib/brand/builder/zone/platform.xml

```diff
@@ -0,0 +1,181 @@
+	<global_mount special="/lib" directory="/lib"
+	    opt="ro,nodevices" type="lofs" />
+	<global_mount special="%P/manifests/joyent"
+	    directory="/lib/svc/manifest"
+	    opt="ro,nodevices" type="lofs" />
+	<global_mount special="%R/site" directory="/lib/svc/manifest/site"
+	    opt="nodevices" type="lofs" />
+	<global_mount special="/sbin" directory="/sbin"
+	    opt="ro,nodevices" type="lofs" />
+	<global_mount special="/usr" directory="/usr"
+	    opt="ro,nodevices" type="lofs" />
+	<global_mount special="%R/local" directory="/usr/local"
+	    opt="nodevices" type="lofs" />
+	<global_mount special="%R/ccs" directory="/usr/ccs"
+	    opt="nodevices" type="lofs" />
+
+	<!-- Local filesystems to mount when booting the zone -->
+	<mount special="/proc" directory="/proc" type="proc" />
+	<mount special="ctfs" directory="/system/contract" type="ctfs" />
+	<mount special="mnttab" directory="/etc/mnttab" type="mntfs" />
+	<mount special="objfs" directory="/system/object" type="objfs" />
+	<mount special="lxproc" directory="/system/lxproc" type="lxproc" />
+	<mount special="swap" directory="/etc/svc/volatile" type="tmpfs" />
+	<mount special="sharefs" directory="/etc/dfs/sharetab" type="sharefs" />
+
+	<!-- Devices to create under /dev -->
+	<device match="arp" />
+	<device match="bpf" />
+	<device match="conslog" />
+	<device match="cpu/self/cpuid" />
+	<device match="crypto" />
+	<device match="cryptoadm" />
+	<device match="dsk" />
+	<device match="dtrace/*" />
+	<device match="dtrace/provider/*" />
+	<device match="eventfd" />
+	<device match="fd" />
+	<device match="full" />
+	<device match="inotify" />
+	<device match="ipnet" />
+	<device match="kstat" />
+	<device match="lo0" />
+	<device match="lofictl" />
+	<device match="lofi" />
+	<device match="log" />
+	<device match="logindmux" />
+	<device match="nsmb" />
+	<device match="net/*" />
+	<device match="null" />
+	<device match="openprom" arch="sparc" />
+	<device match="poll" />
+	<device match="pool" />
+	<device match="ptmx" />
+	<device match="pts/*" />
+	<device match="random" />
+	<device match="rdsk" />
+	<device match="rlofi" />
+	<device match="rmt" />
+	<device match="sad/user" />
+	<device match="smbsrv" />
+	<device match="smbsrv1" />
+	<device match="signalfd" />
+	<device match="svvslo0" />
+	<device match="svvslo1" />
+	<device match="svvslo2" />
+	<device match="svvslo3" />
+	<device match="swap" />
+	<device match="sysevent" />
+	<device match="tap" />
+	<device match="tcp" />
+	<device match="tcp6" />
+	<device match="term" />
+	<device match="ticlts" />
+	<device match="ticots" />
+	<device match="ticotsord" />
+	<device match="timerfd" />
+	<device match="tty" />
+	<device match="tun" />
+	<device match="udp" />
+	<device match="udp6" />
+	<device match="urandom" />
+	<device match="viona" />
+	<device match="vnd/*" />
+	<device match="zero" />
+	<device match="zfs" />
+	<device match="zvol/dsk/%P/%z/*" />
+	<device match="zvol/rdsk/%P/%z/*" />
+
+	<!--
```

Comment:
Thank you for this comment!

Reply:
Copied from the bhyve file. Seemed worth keeping around. Glad you agree.